cin_validator/rules/cin2022_23/rule_8839.py

```diff
@@ -43,8 +43,10 @@ def validate(
     # DF_CHECK: APPLY GROUPBYs IN A SEPARATE DATAFRAME SO THAT OTHER COLUMNS ARE NOT LOST OR CORRUPTED. THEN, MAP THE RESULTS TO THE INITIAL DATAFRAME.
     df_check = df.copy()
     # get all the locations where ICPCnotRequired is null or not true (1)
+    # The rule originally asks for true, not 1, but an analyst coded it for 1, so their LA may use 1 and 0 instead, as such, it now check for either.
```

Comment:
Good catch.

A number of rules ask us to check for "true or 1" so we can consider it a pattern at this point.